test/Riok.Mapperly.Tests/Helpers/MapperConfigurationBuilderTest.cs

```diff
@@ -0,0 +1,90 @@
+using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Configuration;
+using Riok.Mapperly.Helpers;
+
+namespace Riok.Mapperly.Tests.Helpers;
+
+public class MapperConfigurationBuilderTest
+{
+    [Fact]
+    public void ShouldMergeMapperConfigurations()
```

Comment:
Couldn't we use Mapperly itself to ensure `MapperConfiguration` and `MapperAttribute` share all attributes by adding a map method in the integration tests and enforcing strict mappings?

Reply:
Mapperly itself is not possible. Added a comment.